src/schema/me/order/index.js

```diff
@@ -0,0 +1,113 @@
+import {
+  GraphQLInt,
+  GraphQLList,
+  GraphQLObjectType,
+  GraphQLString,
+} from "graphql"
+import Artwork from "schema/artwork"
+import EditionSet from "schema/edition_set"
+import Location from "schema/location"
+import Partner from "schema/partner"
+import money from "schema/fields/money"
+import { GravityIDFields } from "schema/object_identification"
+
+export const OrderLineItemType = new GraphQLObjectType({
+  name: "OrderLineItem",
+  fields: () => ({
+    ...GravityIDFields,
+    quantity: {
+      type: GraphQLInt,
+      description: "Quantity of this item",
+    },
+    artwork: {
+      type: Artwork.type,
+      description: "Artwork that is being ordered",
+      resolve: ({ artwork }) => artwork,
+    },
+    edition_set: {
+      type: EditionSet.type,
+      description: "Edition set on the artwork",
+      resolve: ({ edition_set }) => edition_set,
+    },
+    partner: {
+      type: Partner.type,
+      description: "Partner being ordered from",
+      resolve: ({ partner }) => partner,
+    },
+    partner_location: {
+      type: Location.type,
+      description: "Location of the partner",
+      resolve: ({ partner_location }) => partner_location,
+    },
+    shipping_note: {
+      type: GraphQLString,
+      description: "Shipping note from the partner",
+    },
+    sale_conditions_url: {
+      type: GraphQLString,
+      description: "Sale conditions (set by partner)",
+    },
+  }),
+})
+
+export const AddressType = new GraphQLObjectType({
+  name: "Address",
```

Comment:
This is a pretty generic name....remember when stitching is fully enabled these collisions/conflicts are things that need to be manually dealt with.

maybe this should be `OrderAddress`?